content/docs/command-reference/run.md

```diff
@@ -106,6 +106,10 @@ Relevant notes:
   also means that the stage command needs to recreate any directory structures
   defined as outputs every time its executed by DVC.
 
+- Outputs already in the workspace but missing from `dvc.yaml` can be
+  [added to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  without executing it again.
```

Comment:
Same. Maybe we can clarify files vs. outputs a little? And consider adding `<abbr>` if there's no existing output tooltip nearby already.